src/Classes/GetSelectValueFilter.php

```diff
@@ -25,7 +25,7 @@ class GetSelectValueFilter {
      */
     public $filterValue;
     /**
-     * @var \NetSuite\Classes\GetSelectValueFilterOperator
+     * @var \NetSuite\Classes\GetSelectValueFilterOperator::*
```

Comment:
As with other generator changes this is a bit hard to review. The change is basically the difference. It tells static analysis tools the value can be any of the constants on this class instead of the class itself.